README.md

```diff
@@ -23,11 +23,13 @@ enhancements are welcome.
 Running the doc server locally is easy:
 
 ```
-$ git clone git://github.com/domcleal/puppetmodule.info
-$ cd puppetmodule.info
-$ bundle install
-$ rake gems:update
-$ rackup config.ru
+git clone git://github.com/voxpupuli/puppetmodule.info
+cd puppetmodule.info
+bundle config --local path .vendor/
+bundle config --local jobs $(nrpoc)
```

Comment:
```suggestion
bundle config --local jobs $(nproc)
```

Though perhaps it's redundant now? https://bundler.io/v2.4/man/bundle-config.1.html writes:
> jobs (BUNDLE_JOBS): The number of gems Bundler can install in parallel. Defaults to the number of available processors.

Looking through the changelog this is a Bundler 2.2 feature.

Reply:
ah nice, so it's finally obsolete.